build.psm1

```diff
@@ -446,6 +446,31 @@ Fix steps:
         Pop-Location
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-linux`
Default enablement of Experimental Features.On preview builds, Experimental Features are enabled
```powershell
Expected: Feature PSCommandNotFoundSuggestion to be Enabled
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1: line 47
47:             $expFeature.Enabled | Should -BeEnabled -Name $expFeature.Name
```
Default enablement of Experimental Features.On preview builds, Experimental Features are enabled
```powershell
Expected path '/home/vsts/work/1/a/bins/publish/powershell.config.json' to exist, but it did not exist.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1: line 44
44:         (Join-Path -Path $PSHOME -ChildPath 'powershell.config.json') | Should -Exist
```